test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionExecutorTests.cs

```diff
@@ -1,509 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Discussed in person, we simplified this functionality. These have become integration tests now